src/assets/logos/products/vocabulary.svg

```diff
@@ -0,0 +1,18 @@
+    <path d="M135.38 19.786C135.38 15.706 133.952 13.7 132.558 12.51C130.688 10.878 128.784 10.742 127.832 10.742C125.69 10.742 124.432 11.456 123.718 12C123.344 12.272 123.31 12.306 122.766 12.884V4.99597H117.666V29H122.528L122.494 27.13C122.8 27.504 123.106 27.844 123.514 28.15C125.112 29.408 126.948 29.476 127.662 29.476C131.742 29.476 133.68 26.552 134.292 25.294C134.768 24.342 135.38 22.438 135.38 19.786ZM130.28 19.922C130.28 20.126 130.28 22.71 129.328 24.104C129.022 24.58 128.138 25.6 126.54 25.6C126.166 25.6 125.282 25.532 124.432 24.954C123.82 24.512 123.548 24.104 123.412 23.832C122.63 22.472 122.63 20.16 122.63 19.99C122.63 16.012 124.432 14.686 126.472 14.584C129.192 14.414 129.77 16.726 129.974 17.508C130.246 18.596 130.28 19.48 130.28 19.922Z"/>
+    <path d="M153.706 29V11.252H148.504V22.302C148.504 22.778 148.47 23.22 148.3 23.662C148.062 24.308 147.178 25.26 145.512 25.26C144.73 25.26 144.288 25.022 143.982 24.852C143.03 24.274 142.996 23.424 142.996 22.302V11.252H137.726V21.826C137.726 24.002 137.556 26.858 139.528 28.422C140.922 29.51 142.894 29.51 143.302 29.51C145.478 29.51 147.042 28.626 147.688 28.116C147.892 27.946 148.096 27.742 148.47 27.334V29H153.706Z"/>
+    <path d="M162.598 29V4.99597H157.396V29H162.598Z"/>
+    <path d="M181.841 29C181.399 28.354 181.331 27.776 181.331 26.246V15.876C181.331 14.856 181.161 14.38 180.957 13.904C179.767 11.354 176.401 10.776 173.919 10.776C173.511 10.776 172.661 10.776 171.641 10.912C171.029 11.014 169.091 11.286 167.629 12.714C166.303 14.04 166.269 15.57 166.201 16.284H171.437C171.505 15.91 171.607 15.638 171.675 15.502C171.947 14.924 172.661 14.21 174.191 14.21C176.095 14.21 176.197 15.74 176.197 16.352V17.338C172.831 17.95 170.417 18.426 169.091 19.004C168.309 19.344 165.453 20.874 165.453 24.07C165.453 24.92 165.657 25.804 166.031 26.552C166.609 27.742 168.071 29.476 171.267 29.476C173.477 29.476 174.973 28.592 175.619 28.15C175.993 27.912 176.333 27.674 176.707 27.402V29H181.841ZM176.197 20.5C176.197 22.166 176.129 23.832 174.599 24.92C174.361 25.09 173.545 25.634 172.593 25.634C171.505 25.634 170.995 24.988 170.791 24.648C170.621 24.308 170.587 23.934 170.587 23.832C170.587 23.152 170.927 22.404 172.219 21.86C172.423 21.758 172.627 21.69 172.831 21.622L176.197 20.5Z"/>
+    <path d="M195.238 15.672V11.048C194.592 11.116 193.606 11.184 192.178 11.762C191.022 12.238 190.546 12.714 190.104 13.088V11.252H185.038V29H190.24V19.004C190.24 17.678 190.478 16.794 191.872 16.318C192.518 16.08 193.334 15.978 193.504 15.944L195.238 15.672Z"/>
+    <path d="M212 11.252H206.424L203.5 23.968L200.576 11.252H195L199.658 26.28C199.828 26.858 199.998 27.402 200.168 27.946C200.44 28.796 200.542 29.17 200.542 29.646C200.542 30.258 200.134 30.938 198.91 30.938H196.53V35.12H200.304C201.868 35.12 202.888 34.916 204.01 33.76C204.928 32.774 205.336 31.686 205.744 30.428L212 11.252Z"/>
+  </g>
+</svg>
```

Comment:
We should not duplicate these assets. Makes it harder to update. We developed Vocabulary to develop a SSOT for our UI and this goes against that.